src/boot/efi/boot.c

```diff
@@ -743,7 +766,6 @@ static BOOLEAN menu_run(
                                 config->idx_default_efivar = -1;
                                 status = StrDuplicate(L"Default boot entry cleared.");
                         }
-                        refresh = TRUE;
```

Comment:
hmm?

Reply:
With the => highlighting removed, this refresh is not needed anymore.